docs/book/src/developer/dependencies.md

```diff
@@ -1,26 +1,46 @@
 # Modules and tools dependencies
 
-| Package | Module name | Used by | 
-| --- | ----------- | ------- |
+#### CAPI Packages
+| Package | Module name | Used by |
+| ------- | ----------- | ------- |
 | cluster-api | [sigs.k8s.io/cluster-api](https://github.com/kubernetes-sigs/cluster-api) | [go.mod][go.mod1] |
 | cluster-api/test | [sigs.k8s.io/cluster-api/test](https://github.com/kubernetes-sigs/cluster-api/tree/main/test) | [go.mod][go.mod1]  |
 | cluster-api/hack/tools | [sigs.k8s.io/cluster-api/hack/tools](https://github.com/kubernetes-sigs/cluster-api/tree/main/hack/tools) | [hack/tools/go.mod][go.mod2] |
-| controller-runtime | [sigs.k8s.io/controller-runtime](https://sigs.k8s.io/controller-runtime) | [go.mod][go.mod1] |
-| controller-runtime/tools/setup-envtest | [sigs.k8s.io/controller-runtime/tools/setup-envtest](https://sigs.k8s.io/controller-runtime/tools/setup-envtest) | [hack/tools/go.mod][go.mod2] |
-| controller-tools | [sigs.k8s.io/controller-tools](https://sigs.k8s.io/controller-tools) | [hack/tools/go.mod][go.mod2] |
+
+#### IBM Packages
```

Comment:
I think except these ibm packages, other packages are updated with every capi version update right? so wondering if we can bring that message here

Reply:
Yes will add that note.

Reply:
Done.